rapid_pro/synchronise_contacts.py

```diff
@@ -123,6 +123,13 @@ def filter_valid_contacts(contacts):
                 log.warning(f"Found a contact with a telephone number but without a country code; skipping. "
                             f"The RapidPro UUID is '{contact.uuid}'")
                 continue
+
+            #check for safaricom numbers forwarded with a wrong ke country code formating
+            if contact.urns[0].startswith("tel:+1") and len(contact.urns[0]) < 11:
+                log.warning(f"Found a kenyan telephone number without a proper +254 country code; skipping. "
```

Comment:
Can you make less inferences in the log, just in case, e.g. say "Found a telephone starting +1 but less than 11 digits long. This is probably a malformed Safaricom number, skipping".